memory_profiler.py

```diff
@@ -49,6 +49,28 @@ def unicode(x, *args): return str(x)
     has_psutil = False
 
 
+class MemitResult(object):
```

Comment:
No strong feelings, was simply going with how the IPython team opted to implement `TimeitResult`. I'd be happy to make this change if you'd prefer